packages/eslint-plugin/docs/rules/prefer-at.md

```diff
@@ -0,0 +1,101 @@
+---
+description: 'Enforce the use of `array.at(-1)` instead of `array[array.length - 1]`.'
+---
+
+> 🛑 This file is source code, not the primary documentation location! 🛑
+>
+> See **https://typescript-eslint.io/rules/prefer-at** for documentation.
+
+There are two ways to get the last item of the array:
+
+- `arr[arr.length - 1]`: getting an item by index calculated relative to the length of the array.
+- `arr.at(-1)`: getting an item by the `at` method. [Docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at)
+
+`arr.at(-1)` is a cleaner equivalent to `arr[arr.length - 1]`.
+
+This rule supports the following types:
+
+- `string` or `String`
+- `Array`
+- `Int8Array`
+- `Uint8Array`
+- `Uint8ClampedArray`
+- `Int16Array`
+- `Uint16Array`
+- `Int32Array`
+- `Float32Array`
+- `Uint32Array`
+- `Float64Array`
+- `BigInt64Array`
+- `BigUint64Array`
+
+## Options
+
+```ts
+interface Options {
+  ignoreFunctions?: boolean;
+}
+```
```

Comment:
[Request] No need to explicitly spell out `interface Options`, it's auto-generated. See http://localhost:3000/rules/prefer-at locally (you can start the web server with `yarn start` from the monorepo root).

It would be good to have a text explanation of what `ignoreFunctions` does though.